### PR TITLE
Ee/cp/updates to optimised pipeline

### PR DESCRIPTION
**Pull request by cpavia**
Finished spell checking, annotating and updating optimised_engine_pipeline

### CODE REVIEW

File: src/Components/Engine/optimised_engine_pipeline.ipynb
Comment (mic-0-hal) [inline src/Components/Engine/optimised_engine_pipeline.ipynb:1]: I'll have to run through to check functionality, but the additions look great

**Comment by mic-0-hal** [comment]
Love the naming convention =)

File: src/Components/Engine/optimised_engine_pipeline.ipynb
Comment (cpavia) [reply src/Components/Engine/optimised_engine_pipeline.ipynb:1]: Just checking if any amendments are needed?

Comment (mic-0-hal) [reply src/Components/Engine/optimised_engine_pipeline.ipynb:1]: Having trouble with running the pipeline on my machine. Welcome any help with this

Comment (cpavia) [reply src/Components/Engine/optimised_engine_pipeline.ipynb:1]: Maybe try Ryan? I'm not sure why it isn't working. I didn't change any code, just added comments to help explain the pipeline for juniors

**Review by mic-0-hal — APPROVED** [review]
Approved comments

**Pull request merged**